hooks/post/31-mongo_syspath.rb

```diff
@@ -0,0 +1,15 @@
+def install_syspaths
+  `rpm -qa | grep rh-mongodb34-syspaths`
+  if $?.success?
+    logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
+  else
+    logger.info 'rh-mongodb34-syspaths not present, installing.'
+    Kafo::Helpers.execute('yum install -y -q rh-mongodb34-syspaths')
+  end
+end
+
+# Fail if we are not using the Katello/Foreman Proxy Content scenario.
+katello = Kafo::Helpers.module_enabled?(@kafo, 'katello')
+foreman_proxy_content = Kafo::Helpers.module_enabled?(@kafo, 'foreman_proxy_content')
+logger.error 'MongoDB storage engine upgrade is not currently supported for the chosen scenario.' unless katello || foreman_proxy_content
+install_syspaths
```

Comment:
I think this line needs to be inside the if check on module enabled as well. Otherwise it will log the above message and still call this code.